src/collections/lookup-map.ts

```diff
@@ -1,23 +1,42 @@
 import * as near from '../api'
 import { Bytes } from '../utils';
+import { Serializer } from 'superserial';
 
 export class LookupMap {
     readonly keyPrefix: Bytes;
+    readonly serializer: Serializer;
 
-    constructor(keyPrefix: Bytes) {
+    constructor(keyPrefix: Bytes, classes?: {
+        [className: string]: ((new (...args: any[]) => any) | Function);
+    }) {
         this.keyPrefix = keyPrefix
+        this.serializer = new Serializer({classes})
     }
 
     containsKey(key: Bytes): boolean {
         let storageKey = this.keyPrefix + key
         return near.storageHasKey(storageKey)
     }
 
+    containsObjectKey(key: any): boolean {
+        let storageKey = this.keyPrefix + this.serializer.serialize(key)
+        return near.storageHasKey(storageKey)
+    }
```

Comment:
If you keep these methods as is, you should probably point the expanded function at the other to make it more maintainable and avoid duplication. Something like
```suggestion
    containsObjectKey(key: any): boolean {
        return this.containsKey(this.serializer.serialize(key))
    }
```